tigerpath/majors_and_certificates/majors/ECO_2018.json

```
@@ -25,227 +25,261 @@
+                  "min_needed": null,
+                  "explanation": null,
                   "course_list": [
-                    "MAT 201",
-                    "MAT 202"
+                    "ECO 202",
+                    "ORF 245"
                   ]
                 },
                 {
                   "name": null,
-                  "max_counted": 2,
-                  "min_needed": 2,
-                  "description": null,
-                  "explanation": "",
+                  "max_counted": 1,
+                  "min_needed": "ALL",
+                  "explanation": null,
                   "course_list": [
-                    "MAT 203",
-                    "MAT 204"
+                    "POL 345",
+                    "POL 346"
                   ]
                 }
               ]
+            },
+          ]
+        },
+        {
+          "name": "Math",
+          "max_counted": 1,
+          "min_needed": 1,
+          "explanation": "Incoming majors must complete MAT 175 (or equivalent) by the end of their sophomore year and earn a letter grade of C or better.\n\nStudents will only be exempt from the Economics Department’s MAT 175 pre-requisite if they have already taken an acceptable college level course in multivariate calculus and earned a grade of C or better. Please note that the AP BC calculus curriculum is not sufficient since it only teaches univariate calculus, while MAT 175 focuses on multivariate calculus.\n\nStudents who wish to take math-track econometrics (ECO 312), upper level finance certificate courses (such as ECO 462, ECO 465 and ECO 466), or pursue graduate studies in economics and finance should take MAT 201 and MAT 202 instead of MAT 175. Both courses should be taken for a letter grade by this subset of students. The finance certificate program strongly advises that you take both MAT 201 and MAT 202; only the highest of the two grades is recorded for the purpose of fulfilling the entry requirements. Students who choose to fulfill this requirement by taking the one-semester MAT 175 will be responsible for mastering the linear algebra part of the curriculum on their own.",
+          "completed_by_semester": 4,
+          "req_list": [
+            {
+              "name": null,
+              "max_counted": 1,
+              "min_needed": null,
+              "explanation": null,
+              "course_list": [
+                "MAT 175"
+              ]
+            },
+            {
+              "name": null,
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": null,
+              "course_list": [
+                "MAT 201",
+                "MAT 202"
+              ]
+            },
+            {
+              "name": null,
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": null,
+              "course_list": [
+                "MAT 203",
+                "MAT 204"
+              ]

```

Aren't students allowed to take a combination like MAT 203 with MAT 202?
That should be fine, right?
```suggestion
            {
              "name": null,
              "max_counted": 1,
              "min_needed": "ALL",
              "explanation": null,
              "req_list": [
                {
                  "name": null,
                  "max_counted": 1,
                  "min_needed": null,
                  "explanation": null,
                  "course_list": [
                    "MAT 201",
                    "MAT 203",
                  ]
                },
                {
                  "name": null,
                  "max_counted": 1,
                  "min_needed": null,
                  "explanation": null,
                  "course_list": [
                    "MAT 202",
                    "MAT 204"
                  ]
                }
              ]
```